Sources/Networking/Modifiers/Processors/EndpointRequestStorageProcessor/EndpointRequestStorageProcessor.swift

```diff
@@ -25,7 +25,15 @@ open class EndpointRequestStorageProcessor: ResponseProcessing, ErrorProcessing
 
     private lazy var responsesDirectory = fileManager.temporaryDirectory.appendingPathComponent("responses")
     private lazy var requestCounter = Counter()
-
+    private lazy var multipeerConnectivityManager: MultipeerConnectivityManager? = {
```

Comment:
The thing I am worried about is that the `getAllStoredModels()` function might get pretty computationally heavy in case we have hundreds of stored requests and we might not be even using the multipeer macos up during development, so we would either need to pass a parameter to let the `EndpointRequestStoreProcessor` know wether we want to initialise the `MultipeerConnectivityManager` or as you suggested we could create a new processor that would separate the multipeer connectivity functionality, although it might include a lot of duplicated code from this processor 🤔

Reply:
I would inject some flag/configuration to decide whether to use multipeer to send data and to decide if to send whole history or just current session. 
About all stored data - maybe we should add some cleanup mechanism somewhere, not necessarily here

Reply:
I've added Config options to handle both of these requirements.